### ISSUE TITLE
Proposal: django-admin-honeypot

### ISSUE THREAD

**Issue by 9mido** [jazzband/help#242]
https://github.com/dmpayton/django-admin-honeypot

Project Details

django-admin-honeypot

Project Description

django-admin-honeypot is a fake Django admin login screen to log and notify admins of attempted unauthorized access.

Project Leads
@dmpayton

Confirmation of Guidelines

Please confirm the following:

I am not the project owner, but dmpayton who is the owner seemed to have abandoned this project. See https://github.com/dmpayton/django-admin-honeypot/issues/66

I just wanted to help move things along to possibly move this project over to jazzband. I do not want to maintain django-admin-honeypot myself as I am not experienced enough.

This project is a viable candidate for Jazzband.

This project fulfills the documentation requirements.

This project is tested.

I cannot speak for dmpayton but I would imagine he would be happy to adopt Jazzband's code of conduct in this project.

I cannot speak for dmpayton but I would imagine that he is happy to include Jazzband's contributing guidelines in this project.

**Comment by MRigal** [comment]
@9mido I agree this project should be kept maintained. I could help out there

**Comment by 9mido** [comment]
@MRigal That sounds great. What are the next steps for you or someone else to take over django-admin-honeypot?

**Comment by nikosmichas** [comment]
Hello! Is there any update on this? This is a widely used library that needs some work in order to be compatible with Django 4.0

**Comment by jezdez** [comment]
Hi all, this seems to be a good candidate for sure, but @dmpayton would need to agree to transfer it to Jazzband, which I haven't seen here. There seems to be some commit activity upstream now as well.

@dmpayton What's your take on this?

**Comment by MRigal** [comment]
Indeed @jezdez , @dmpayton seems to have get back to work recently and fixed the most urgent issues. Do you prefer to keep it yours or would you mind delegating it to jazzband?
